index.bs

```diff
@@ -1491,13 +1495,25 @@ underlined text, and a <a>WebVTT cue span end tag</a> "<code>u</code>".</p>
  <li>
   One or more occurrences of the following group of components, in the order given:
   <ol>
-   <li><a>WebVTT cue internal text</a>, representing the ruby base.</li>
-   <li>A <a>WebVTT cue span start tag</a> "<code>rt</code>" that disallows an annotation.</li>
-   <li>A <dfn>WebVTT cue ruby text span</dfn>: <a>WebVTT cue internal text</a>, representing the
-   ruby text component of the ruby annotation.</li>
-   <li>A <a>WebVTT cue span end tag</a> "<code>rt</code>". If this is the last occurrence of this
-   group of components in the <a>WebVTT cue ruby span</a>, then this last end tag string may be
-   omitted.</li>
+   <li><a>WebVTT cue internal text</a>, representing a ruby base.</li>
+   <li>a ruby text component:</li>
+   <ol>
+    <li>A <a>WebVTT cue span start tag</a> "<code>rt</code>" that disallows an annotation.</li>
+    <li>A <dfn>WebVTT cue ruby text span</dfn>: <a>WebVTT cue internal text</a>, representing the
+    ruby text component of the ruby annotation.</li> <li>A <a>WebVTT cue span end tag</a>
+    "<code>rt</code>". If this is the last occurrence of this group of components in the <a>WebVTT
+    cue ruby span</a> or a <a>WebVTT cue span start tag</a> "<code>rb</code>" is next, then this
+    last end tag string may be omitted.</li>
+   </ol>
+   <li>a ruby base text component:</li>
+   <ol>
+    <li>A <a>WebVTT cue span start tag</a> "<code>rb</code>" that disallows an annotation.</li>
+    <li>A <dfn>WebVTT cue ruby base text span</dfn>: <a>WebVTT cue internal text</a>, representing
+    the ruby base text component of the ruby annotation.</li>
+    <li>A <a>WebVTT cue span end tag</a> "<code>rb</code>". If this is the last occurrence of this
+    group of components in the <a>WebVTT cue ruby span</a> or a <a>WebVTT cue span start tag</a>
+    "<code>rt</code>" is next, then this last end tag string may be omitted.</li>
+   </ol>
```

Comment:
This appears to be wrong... It allows `<ruby>foo<rt>bar<rb>baz</ruby>`, but not `<ruby><rb>foo<rt>bar</ruby>`